C/services/south/south.cpp

```diff
@@ -22,6 +26,7 @@
 #include <filter_plugin.h>
 
 extern int makeDaemon(void);
+static int createTimerFd(unsigned int usecs);
```

Comment:
Why a static function and not a private method

Reply:
It's a helper function, which I feel could be used by other code also in future. It's not specific to SouthService class, hence didn't want to make it part of this class.

Reply:
@MarkRiddoch - Please let me know if I should make this change or leave it as it is.

Reply:
It's up to you, personally I would make it a member function, but I don't think it is that much of an issue that I would stop the checkin